server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseServicesImpl.java

```diff
@@ -121,4 +139,59 @@ private boolean isSubordinateTo(UUID reportMember, UUID currentUserId) {
         return memberProfileServices.getSubordinatesForId(currentUserId)
                 .stream().anyMatch(member -> member.getId().equals(reportMember));
     }
+
+    public void sendPulseLowScoreEmail(PulseResponse pulseResponse) {
+        if (pulseResponse == null) return;
+
+        boolean hasLowInternalScore = pulseResponse.getInternalScore() != null && pulseResponse.getInternalScore() <= 2;
+        boolean hasLowExternalScore = pulseResponse.getExternalScore() != null && pulseResponse.getExternalScore() <= 2;
+
+        System.out.printf("%b %b\n%n", hasLowExternalScore, hasLowInternalScore);
+        if (!(hasLowInternalScore || hasLowExternalScore)) return;
+
+        UUID teamMemberId = pulseResponse.getTeamMemberId();
+        MemberProfile surveyTakerMemberProfile;
+        if (teamMemberId != null && memberRepo.existsById(teamMemberId)) {
+            surveyTakerMemberProfile = memberProfileServices.getById(teamMemberId);
+        } else {
+            log.warn("profileTaker does not exist");
```

Comment:
Updated if we go with capitalization suggestion above

```suggestion
            LOG.warn("profileTaker does not exist");
```